### ISSUE TITLE
Not adding balance of unspent outputs (assets)

### ISSUE THREAD

**Issue by off-bank** [Colored-Coins/Colored-coins-add-on-for-Copay#17]
When more of the same asset is received, it doesn't add the balance to that asset, it creates more rows of the same asset instead. 
I sent 10 units + 11 units of the same asset. Instead of 21 units in one row (under "Assets" page), I have 2 rows of the same asset. If I open one of them and try to send 12 units, it doesn't allow me, limiting the transfer to the individual balance (10 or 11).

**Comment by troggy** [comment]
Yeah, that's known issue. Right now it shows assets by utxo (one row per utxo).
